.github/workflows/test-emacs-plugin.yml

```diff
@@ -0,0 +1,47 @@
+# Copyright (C) 2020  Matthew Glazar
+# See end of file for extended copyright information.
+
+name: test Emacs plugin
+on:
+  push:
+  pull_request:
+    types: [opened, synchronize]
+
+jobs:
+  build:
+    name: Emacs
+    runs-on: ubuntu-latest
+    container: ghcr.io/quick-lint/quick-lint-js-github-builder:v1
```

Comment:
This version of the container doesn't include your changes. I'll upload a version for you to test.

Reply:
I published `ghcr.io/quick-lint/quick-lint-js-github-builder:v2-wip` which has emacs installed (per your Dockerfile).